### ISSUE TITLE
Table auto-rotate crashing with `get` [ Windows ]

### ISSUE THREAD

**Issue by notryanb** [nushell/nushell#1532]
**Describe the bug**
The new auto-rotate feature for single row tables is working, however when trying to `get` one of the fields, nushell returns the value and immediately crashes with an `Io(Os { code: 87, kind: Other, message: "The parameter is incorrect." })` message.

The frequency at which this occurs is hard to pin down. Sometimes it happens immediately, sometimes it takes several commands. `pick col_name` works without any issues.

**To Reproduce**
1.  `echo $nu.env` or `ls | first` to get an auto-rotated table 
2.  `get` one of the values. ex `ls | first | get name` several times.

**Expected behavior**
The value is returned with no crashing.

**Screenshots**
```
nushell on 📙 master is 📦 v0.11.1 via 🦀 v1.42.0
❯ ls | first
──────────┬─────────────
 name     │ .azure
 type     │ Dir
 size     │
 modified │ 2 weeks ago
──────────┴─────────────
nushell on 📙 master is 📦 v0.11.1 via 🦀 v1.42.0
❯ ls | first | get name
.azure
nushell on 📙 master is 📦 v0.11.1 via 🦀 v1.42.0
❯ ls | first | get name
.azure
nushell on 📙 master is 📦 v0.11.1 via 🦀 v1.42.0
❯ ls | first | get name
.azure
Error: Io(Os { code: 87, kind: Other, message: "The parameter is incorrect." })
```

another example:

```
nushell on 📙 master is 📦 v0.11.1 via 🦀 v1.42.0
❯ echo $nu.env | pick NUMBER_OF_PROCESSORS
──────────────────────┬───
 NUMBER_OF_PROCESSORS │ 8
──────────────────────┴───
nushell on 📙 master is 📦 v0.11.1 via 🦀 v1.42.0
❯ echo $nu.env | get NUMBER_OF_PROCESSORS
8
Error: Io(Os { code: 87, kind: Other, message: "The parameter is incorrect." })
```

**Configuration (please complete the following information):**
 - OS: Windows Build 1903
 - Version: 0.11.1
 - Optional features: Both with and without `--features=stable`

**Add any other context about the problem here.**
This works as expected on my Mac machine.

I am also experiencing a single test failure on my Window machines with `cargo test --all --features stable,test-bins`, which may be related to this problem?

```
failures:

---- shell::pipeline::commands::internal::takes_rows_of_nu_value_strings_and_pipes_it_to_stdin_of_external stdout ----
=== stderr

thread 'shell::pipeline::commands::internal::takes_rows_of_nu_value_strings_and_pipes_it_to_stdin_of_external' panicked at 'assertion failed: `(left == right)`
  left: `""`,
 right: `"AndKitKat"`', tests\shell\pipeline\commands\internal.rs:31:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    shell::pipeline::commands::internal::takes_rows_of_nu_value_strings_and_pipes_it_to_stdin_of_external

test result: FAILED. 35 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out
```

**Comment by jonathandturner** [comment]
I wonder if this has the same root cause as https://github.com/nushell/nushell/issues/1491

**Comment by jonathandturner** [comment]
I can't repro these test cases with the fix for #1491. Closing, but please re-open if you can still cause this to happen with the latest master.
